src/lib/task.ts

```diff
@@ -59,9 +59,11 @@ export class Task<Ctx, Renderer extends ListrRendererFactory> extends Subject<Li
   public renderHook$: Subject<void>
 
   public prompt: undefined | PromptInstance | PromptError
-  private enabled: boolean
-  private enabledFn: ListrTask<Ctx, Renderer>['enabled']
-
+  public exitOnError!: boolean
```

Comment:
I have changed this in the last version, I have to merge this part manually just note to self.